mappings/net/minecraft/entity/mob/EntityMob.mapping

```diff
@@ -2,18 +2,25 @@ CLASS none/sq net/minecraft/entity/mob/EntityMob
 	CLASS none/sq$a
 	FIELD a MOB_FLAGS Lnone/kj;
 	FIELD bA persistent Z
+	FIELD bB pathNodeTypeWeights Ljava/util/Map;
 	FIELD bC lootTable Lnone/kp;
 	FIELD bD lootTableSeed J
 	FIELD bE leashed Z
 	FIELD bF holdingEntity Lnone/sg;
 	FIELD bG leashTag Lnone/dt;
+	FIELD b look Lnone/tn;
+	FIELD br goalSelector Lnone/ub;
+	FIELD bs targetSelector Lnone/ub;
 	FIELD bt handDropChances [F
 	FIELD bu armorDropChances [F
 	FIELD bv target Lnone/sp;
 	FIELD bw visibilityCache Lnone/wb;
 	FIELD bx handItems Lnone/dc;
 	FIELD by armorItems Lnone/dc;
 	FIELD bz pickUpLoot Z
+	FIELD f move Lnone/to;
+	FIELD g jump Lnone/tm;
+	FIELD h navigation Lnone/vw;
```

Comment:
Are these (and `look`) AI tasks or something? I feel like prefixing them would make sense. `taskLook` etc?